content/blog/pko-2-0-ga/index.md

```diff
@@ -0,0 +1,73 @@
+    linkedin: "Announcing Pulumi Kubernetes Operator 2.0 GA: Now with enhanced logging control, richer controller events, and flexible workspace management. Experience enterprise-grade infrastructure automation with improved observability and resource management in your Kubernetes clusters."
+
+---
+
+Today marks an exciting milestone as we announce the General Availability (GA) release of the Pulumi Kubernetes Operator 2.0! This release builds upon the foundation we [laid during the beta phase](/blog/pulumi-kubernetes-operator-2-0/), delivering a production-ready solution that transforms how teams manage their cloud infrastructure.
+<!--more-->
+
+[Learn more about the Pulumi Kubernetes Operator in our documentation](/docs/iac/using-pulumi/continuous-delivery/pulumi-kubernetes-operator)
+
+## What Is the Pulumi Kubernetes Operator?
+
+The Pulumi Kubernetes Operator brings infrastructure automation directly to your Kubernetes cluster. By treating your infrastructure as native Kubernetes resources, it enables teams to manage their entire cloud environment—from databases to DNS records—using familiar Kubernetes workflows and tools.
+
+Imagine having a dedicated infrastructure automation engine running within your cluster, continuously ensuring your cloud resources stay in sync with your desired state. That's exactly what the Pulumi Kubernetes Operator delivers, complete with the observability and control you need for production environments.
+
+## The Evolution to 2.0
+
+Pulumi Kubernetes Operator 2.0 introduces a completely new architecture for running Pulumi programs. The key changes include:
+
+- Each Stack now runs in its own dedicated workspace pod, rather than within the Operator pod
+- A new Workspace Custom Resource manages the workspace pod lifecycle
+- Stack operations are coordinated through a new Update Custom Resource
+- The Stack Custom Resource API remains the primary interface, maintaining backward compatibility
+
+This architecture delivers improved resource isolation, better secrets management, and flexible workspace customization options. The pod-per-stack model ensures reliable resource allocation and clear operational boundaries for each deployment.
+
+## What's New in GA?
+
+The GA release introduces three powerful capabilities that give you more control over your deployments:
+
+### Enhanced Logging Control
+
+You can now set the log verbosity level of the Pulumi CLI for any given stack or workspace. This feature offers granular control over logging output, making debugging and fine-tuning your deployment process more efficient.
+
+### New Controller Events for Better Instrumentation
+
+Additional controller events have been introduced to provide richer instrumentation and observability. These events offer improved insights into the operator's behavior, enabling more effective monitoring of your infrastructure.
+
+### Workspace Reclaim Policy
+
```

Comment:
Use `workspaceReclaimPolicy`.
The supported policies are 'retain' and 'delete', to cleanup the workspace once the stack is fully deployed.

Reply:
More specifically, the first letter should be capitalized: `Retain` and `Delete`.